cmd/microcloud/ask.go

```diff
@@ -76,24 +76,25 @@ func (c *initConfig) askUpdateProfile(profile api.ProfilesPost, profiles []strin
 }
 
 // askRetry will print all errors and re-attempt the given function on user input.
-func (c *initConfig) askRetry(question string, f func() error) {
+func (c *initConfig) askRetry(question string, f func() error) error {
 	for {
 		retry := false
 		err := f()
 		if err != nil {
-			fmt.Println(err)
+			if err.Error() != "" {
```

Comment:
Actually, I got rid of the `fmt.Errorf("")` pattern I mentioned to you.

I left this check in because if the error is ever empty for some reason, we'll print a new line with `fmt.Println`.

Reply:
But wouldn't it be better to print the newline instead of vanishing the error? Then you have at least an indicator (the newline) that something is off. 

To me it sounds like an empty error would be something you don't want to have so we should not silently suppress it.

Reply:
Sure, I'll remove the check.